lib/LedgerSMB/Scripts/login.pm

```diff
@@ -134,7 +135,7 @@ sub authenticate {
                      [ 'Content-Type' => 'text/plain; charset=utf-8' ],
```

Comment:
@ziuq for now, leave all (not yet done) changes to 454 status codes.
I'll work on that later.